spec.html

```diff
@@ -27261,10 +27362,10 @@ <h1>Source Text Module Records</h1>
                 [[ImportName]]
               </td>
               <td>
-                a String or ~namespace-object~
+                a String or ~namespace-object~ or ~source~
               </td>
               <td>
-                The name under which the desired binding is exported by the module identified by [[ModuleRequest]]. The value ~namespace-object~ indicates that the import request is for the target module's namespace object.
+                The name under which the desired binding is exported by the module identified by [[ModuleRequest]]. The value ~namespace-object~ indicates that the import request is for the target module's namespace object. The value ~source~ represents an import of the import phase representation at that phase name.
```

Comment:
The added sentence reads strangely generic. Please reword to just talk about the source phase.